deployment.rst

```diff
@@ -167,6 +167,16 @@ as you normally do:
     using :ref:`Symfony Flex <symfony-flex>`) before running this command so
     that the ``post-install-cmd`` scripts run in the ``prod`` environment.
 
+.. tip::
+
+    To improve performance and if you have Symfony Flex 1.2 or later installed, you may
+    leverage a (feature)[https://symfony.com/doc/4.4/configuration.html#configuring-environment-variables-in-production]
+    that dumps your environments variables into a flat file:
+
+.. code-block:: terminal
+
+    $ composer dump-env prod
```

Comment:
As this belongs to the tipp, it should have 4 spaces more indention

Reply:
@OskarStark you mean like now? is it just this line or also the above `.. code-block:: terminal` ?
thx